lib/thredded.rb

```diff
@@ -143,7 +143,7 @@ def self.user_class=(user_class_name)
     unless user_class_name.is_a?(String)
       fail "Thredded.user_class must be set to a String, got #{user_class_name.inspect}"
     end
-    @@user_class_name = user_class_name # rubocop:disable Style/ClassVars
+    @@user_class_name = user_class_name.demodulize # rubocop:disable Style/ClassVars
```

Comment:
The problem was because I didn't override _standalone.html.erb which does use Thredded.user_class_name.underscore. Overriding this and removing the underscore calls in the initializer made it work without the change.

So an actual fix may be to add demodulize to these two places.